shared/js/background/atb.js

```diff
@@ -161,6 +168,67 @@ const ATB = (() => {
             return new URLSearchParams()
         },
 
+        /**
+         * Takes an optional date object and returns an object with ATB majorVersion, minorVersion, and version string
+         * The majorVersion rolls over every Wed at 00:00:00 ET
+         * The minorVersion rolls over every night at midnight ET
+         *
+         * @typedef {Object} AtbVersionInfo
+         * @property {number} majorVersion - # of weeks since noon EST on 2/24/16
+         * @property {number} minorVersion - # of days into the current week
+         * @property {string} version - the version string combining both major and minor version. Format: `v${majorVersion}-${minorVersion}`
+         *
+         * @return {AtbVersionInfo}
+         */
+        getATBVersionInfo: () => {
```

Comment:
I've ported `getATBVersionInfo` from the static-pages repo as I didn't find it in the extension codebase.

Reply:
We *currently* just pull it from the server instead using the tab URLs